app/src/main/java/org/covidwatch/android/presentation/home/HomeViewModel.kt

```diff
@@ -78,7 +78,7 @@ class HomeViewModel(
                 _turnOnBluetoothAction.value = Event(Unit)
             }
             is BannerAction.PotentialRisk -> {
-                // TODO: navigate to Potential Risk screen
+                _userFlow.value = PotentialRisk
```

Comment:
I don't this that this is related to UserFlow. UserFlows are different ways of launching app from home screen, hence: FirstTime, Setup, ReturnUser (this names are from figma). UserFlows are also needed to handle ui-state of home screen

In your case we can create separate liveData, that will handle navigation from homeFragment to potentialRiskFragment:

```kotlin
private val _potentialRiskAction = MutableLiveData<Event<Unit>>()
val potentialRiskAction: LiveData<Event<Unit>> get() = _potentialRiskAction
```

you can read here why we need to wrap it in `Event`: 
https://medium.com/androiddevelopers/livedata-with-snackbar-navigation-and-other-events-the-singleliveevent-case-ac2622673150

Reply:
Done.